library.rb

```diff
@@ -0,0 +1,218 @@
+require "date"
+
+class Library
+  attr_reader :books, :authors
+  def initialize
+    @books = []
+    @authors = []
+    @orders = []
+    @readers = []
+
+  end
+
+  def add_book(book)
+    @books << book
+  end
+
+  def add_author(author)
+    @authors << author
+  end
+
+  def add_order(order)
+    @orders << order
+  end
+
+  def add_reader(reader)
+    @readers << reader
+  end
+
+  def to_s
+    "Books:\n #{@books.map(&:to_s).join("\n")}\n"\
+    "Authors:\n #{@authors.map(&:to_s).join("\n")}\n"\
+    "Orders:\n #{@orders.map(&:to_s).join("\n")}\n"\
+    "Readers:\n #{@readers.map(&:to_s).join("\n")}"
+  end
+
+  def save_from_file
+    File.open("to_s.txt", "w+") do |f|
```

Comment:
create a constant with file name where library should be saved

Reply:
done [here](https://github.com/ArtemVlasenko/Library/pull/1/commits/7393ff7c29112fbdbc98ec99ca416aa8d3eccbad)